package.json

```diff
@@ -116,7 +115,6 @@
     "husky": "0.14.3",
     "jest-canvas-mock": "1.0.2",
     "jest-localstorage-mock": "2.2.0",
-    "jest-puppeteer": "3.0.1",
```

Comment:
Any reason to remove this dependency in this PR?

Reply:
Sorry, I was making tests and this dependencies were blocking me. I've rebased and now this is gone from this PR.